posts/inside-rust/2020-03-19-all-hands-retrospective.md

```diff
@@ -0,0 +1,146 @@
+soon have a bank account for collecting funds, and we started collecting
+information to repay people who had booked travel.
+
+In retrospect, it's clear that this was premature. It seemed like all the
+pieces were in place, and we knew that if people didn't start booking tickets,
+they never would. But the pieces were not in place, not actually, and we clearly
+should have taken this moment ask whether we were really going to be able to
+pull this off.
+
+### Realising it won't work.
+Finally, this all started to unravel in a about a month away from the event.
+In short, we encountered some logistical hurdles, which we won't go into, but
+which also prompted us to review the finances in more detail. This revealed
+that our estimates were off. Thus, we were faced with a tough decision:
+
+* If we continue with the event, there was the chance that the logistics would
+  just fail to come together. That would be a true disaster.
+* Further, we might not be able to raise enough money to be able to pay for
+  the entirety of people's travel and lodging as we had promised. This would be
+  unfortunate, though perhaps something people could live with.
+* On the other hand, if we cancel, we have to deal with the fact that people
+  had already started booking travel.
+
+After much analysis and discussion with the Rust core team, we ultimately
+opted for the painful but ultimately safer route: cancelling the event.
+
+**Addendum** Since we cancelled the All Hands, outbreak of the [COVID-19] in
+Europe in the recent weeks, would have caused the event to be cancelled
+regardless. In a weird twist of fate, we were fortunate to cancel when we did.
+
+[covid-19]: https://en.wikipedia.org/wiki/2019–20_coronavirus_pandemic
+
+## Lessons Learned
+**Bound uncertainty and total cost.** — Promising to cover all travel with
+no kind of upper limit was unwise. It had worked in the past, perhaps, but the
+event was growing, and that was partly luck.
+
+**Location, location, location** — It was hard to find a good location. While
+Thessaloníki was cheap and great in some ways, we encountered some logistical
+hiccups that made it less than ideal. For example; we found that the cost of a
+lot of the flights for our international members were quite high compared to
+other European
+cities.
+
+**Know your audience** — Some companies weren't interested in sponsoring
+because the event wasn't sufficiently high profile, or because the "ROI"
+("Return On Investment") didn't quite fit into the budgets and categories it
+would have to come from. We also had some uncertainty about how much
+"participation" to offer in exchange for sponsorship, since we didn't want to
+give the impression that one could just "buy your way" into the All Hands — at
+the same time, there is clear value to the Rust organisation in having
+sponsors participate.
+
+## The Future
+
+Right now, we're currently organising reimbursing anyone who had purchased non
```

Comment:
I think we should add a sentence crediting Mozilla for paying the reimbursements here (or somewhere).